modules/next/next.routing.yml

```diff
@@ -14,5 +14,6 @@ next.validate_preview_url:
     _controller: 'Drupal\next\Controller\NextPreviewUrlController::validate'
   methods: [ POST ]
   requirements:
+    # Handling a validation using dependency injection.
```

Comment:
It should be "Handling validation using dependency injection."

Reply:
Done ~~ Thanks